tests/python-example-auth-scram-sha-1.sh

```diff
@@ -2,6 +2,11 @@
 
 set -ex
 
-pip3 install pymongo
+pip install --user pipenv
+
+pipenv --python $(python3 -c 'import sys; print(".".join(map(str, sys.version_info[:3])))')
+
+. $(pipenv --venv)/bin/activate
```

Comment:
I'm not sure if `.` works correctly there. I think `python3` below would use a system Python, not the one from the virtualenv.

I think we could simplify that by using https://docs.python.org/3/library/venv.html